addons/compat_cup_vehicles/CfgVehicles.hpp

```diff
@@ -0,0 +1,53 @@
+class CfgVehicles {
+    class CUP_MTVR_Base;
+    class CUP_MTVR_Reammo_Base: CUP_MTVR_Base {
+        transportAmmo = 0;
```

Comment:
I think we can remove rearm/repair of this as it's done dynamically now

Reply:
`GVAR(defaultSupply)` should stay unless it inherits the same values.

`transportX` and `GVAR(canRepair)` (if CUP has it configured properly) can be removed.